src/shiver/models/histogram.py

```diff
@@ -59,6 +60,18 @@ def connect_error_message(self, callback):
         """Set the callback function for error messages"""
         self.error_callback = callback
 
+    def symmetry_operations(self, symmetry):
+        """Validate the symmetry value with mandit"""
+        if len(symmetry) != 0:
+            try:
```

Comment:
I was looking at how MDNorm [checks the symmetries and it allows more possibles](https://github.com/mantidproject/mantid/blob/b37778cec0dd3d39dbd0516723760c1e6746c354/Framework/MDAlgorithms/src/MDNorm.cpp#L352-L365)

So we should also check first 
```python
from mantid.geometry import SpaceGroupFactory, PointGroupFactory
if (symmetry.isnumeric() and SpaceGroupFactory.isSubscribedSymbol(int(symmetry))) or SpaceGroupFactory.isSubscribedSymbol(symmetry) or PointGroupFactory.isSubscribed(symmetry):
    # then it's valid
else:
    # check with SymmetryOperationFactory.createSymOps
```

You can get a list of all valid inputs for testing by looking at
```python
print(SpaceGroupFactory.getAllSpaceGroupSymbols())
print(SpaceGroupFactory.getAllSpaceGroupNumbers())
print(PointGroupFactory.getAllPointGroupSymbols())
```